pixiedust/display/chart/renderers/templates/renderer.html

```diff
@@ -30,6 +30,17 @@
                         data-default={{this.options.get(option.name,None) or option.metadata.default}}
                         style="margin: 0 0.6em;"></div>
                 </div>
+                {% if option.name == 'numbins' and this.get_numbins_options_pixieapp() %}
```

Comment:
I would prefer if we don't hardcode specific cases in the generic renderer.html. I suggest we define a new type of common options like pd_app that would generate a button pretty much the same way you're doing it now.